assets/js/users.js

```diff
@@ -1,11 +1,33 @@
-/* global userRepository */
-
-function User(firstName, lastName, emailAddress, username, password, avatar) {
-  this.id = 1;
+function User(id, firstName, lastName, emailAddress, username, password, avatar) {
+  this.id = id;
   this.firstName = firstName;
   this.lastName = lastName;
   this.emailAddress = emailAddress;
   this.username = username;
   this.password = password;
   this.avatar = avatar;
 }
+
+fetch('https://jsonplaceholder.typicode.com/users')
+  .then(response => response.json())
+  .then(data => {
+    data.forEach(user => {
+      var names = user.name.split(' ');
+      var firstName = names[0];
+      var lastName = names.slice(1).join(' ');
+      var usernameInLowercase = user.username.toLowerCase();
+
+      var newUser = new User(
+        user.id,
+        firstName,
+        lastName,
+        user.email,
+        usernameInLowercase,
+        'temporaryPassword',
+        'https://i.pravatar.cc/300'
```

Comment:
Please see my comments in the Project board [https://github.com/users/Vickysug/projects/1/views/1?pane=issue&itemId=51889510](https://github.com/users/Vickysug/projects/1/views/1?pane=issue&itemId=51889510) for creating a permenent Id for avatar.